transactor/src/main/scala/io/mediachain/transactor/JournalServer.scala

```diff
@@ -50,11 +57,45 @@ object JournalServer {
     val server = Server.build(address, copycatdir, datastore, sslConfig)
     
     if (cluster.isEmpty) {
-      server.bootstrap().join()
+      server.bootstrap.join()
     } else {
       server.join(cluster.map {addr => new Address(addr)}).join()
     }
     
+    if (interactive) {
+      println("Running with interactive console")
+      interactiveLoop(server)
+    } else {
+      serverLoop(server)
+    }
+    datastore.close()
+  }
+  
+  def serverLoop(server: CopycatServer) {
     while (server.isRunning) {Thread.sleep(1000)}
   }
+
+  def interactiveLoop(server: CopycatServer) {
```

Comment:
I would have probably just trapped SIGINT = shutdown and SIGHUP = leave but this is fine